### DISCUSSION TITLE
Lavalink version?

### DISCUSSION THREAD

**Discussion by holedaemon** [jareddantis/blanco-bot#46] [Q&A] [answered]
Hey there, thanks for your work here, seems solid. I'm trying to run blanco for some friends and myself, but I'm running into some trouble. I've got blanco running alongside a Lavalink instance, but when I try to run commands, Mafic is throwing warnings saying the version of the Lavalink node is unknown and the response is 400. So I'm just curious, what specific version of Lavalink do you use to run your instance?

Thanks

**Comment by jareddantis** [comment, marked as answer]
Hello! Blanco is designed for use with Lavalink v4. It's currently in beta but it's solid enough for daily driving.

Originally Blanco used Lavalink.py which worked with Lavalink v3, but a couple months ago there was a bout of 403 errors with YouTube playback that was patched with the v4 beta. Lavalink.py didn't support v4 at the time so I had to quickly rewrite Blanco for Mafic, which _did_ support v4.

**Reply by holedaemon** [reply]
Awesome, thank you. I appreciate the fast reply. Assuming you're on Docker, what tag are you using for Lavalink? I've tried a few different v4 tags and I'm still getting the unknown version warning. It's possible I have something else misconfigured, just want to cover all my bases though.

Edit: Ha, yeah, my stuff was misconfigured. Seems good now though, thanks again for your help.
